recipes-rubygems/rubygems-foo-bar_1.2.3.bb

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: GPL-2.0-only
+SUMMARY = "RubyGem: activesupport"
+DESCRIPTION = "A toolkit of support libraries and Ruby core extensions extracted from the Rails framework"
+HOMEPAGE = "https://rubyonrails.org"
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://MIT-LICENSE;md5=a2e0e281b9b6c47f722da2255eca5d68"
+
+DEPENDS_class-native += "\
+    rubygems-concurrent-ruby-native \
+    rubygems-i18n-native \
+    rubygems-minitest-native \
+    rubygems-tzinfo-native \
+    rubygems-zeitwerk-native \
+"
+
+RDEPENDS_${PN}_class-target += "\
+    rubygems-concurrent-ruby \
+    rubygems-i18n \
+    rubygems-minitest \
```

Comment:
potential issue: On a multiline assignment, line indent is desirable. Current 4/32